### PR TITLE
Pt-br translation fixes

### PR DESCRIPTION
**Pull request by RobertoCassio**
Fixed some mistakes in the translation in the bundle_pt_BR.properties.

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.data.transfer/OSGI-INF/l10n/bundle_pt_BR.properties [hunk 1]
```diff
@@ -67,7 +67,7 @@ dataTransfer.processor.sql.property.includeAutoGenerated.description
 dataTransfer.processor.sql.property.includeAutoGenerated.label                  = Incluir colunas geradas
 dataTransfer.processor.sql.property.omitSchema.description                      = Omita o nome do esquema / cat\u00E1logo nas instru\u00E7\u00F5es INSERT
 dataTransfer.processor.sql.property.omitSchema.name                             = Omit schema name
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.data.transfer/OSGI-INF/l10n/bundle_pt_BR.properties:69]: `dataTransfer.processor.sql.property.omitSchema.name = Omit schema name`

Hello. Could you translate this also?

File: plugins/org.jkiss.dbeaver.data.transfer/OSGI-INF/l10n/bundle_pt_BR.properties [hunk 2]
```diff
@@ -104,7 +104,7 @@ dataTransfer.producer.stream.processor.csv.property.header.description
 dataTransfer.producer.stream.processor.csv.property.header.name                 = Posi\u00E7\u00E3o do cabe\u00E7alho
 dataTransfer.producer.stream.processor.csv.property.nullString.description      = String pura  usado parar marca valor NULL
 dataTransfer.producer.stream.processor.csv.property.nullString.name             = Marcar valor NULL
-dataTransfer.producer.stream.processor.csv.property.quoteChar.description       = Carastere usado para citar valor 
+dataTransfer.producer.stream.processor.csv.property.quoteChar.description       = Caractere usado para citar valor 
 dataTransfer.producer.stream.processor.csv.property.quoteChar.name              = Caractere de cita\u00E7\u00E3o
 dataTransfer.producer.stream.processor.csv.property.timestampFormat.description = Formato padr\u00E3o Data/Hora 
 dataTransfer.producer.stream.processor.csv.property.timestampFormat.name        = Formato Date/time
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.data.transfer/OSGI-INF/l10n/bundle_pt_BR.properties:110]: `dataTransfer.producer.stream.processor.csv.property.timestampFormat.name = Formato Date/time`

This one also looks partly translated.

**Comment by LonwoLonwo** [comment]
Hello @RobertoCassio 

Thanks for your contribution. 
I added some comments. Please take a look.

**Comment by RobertoCassio** [comment]
You're welcome, i will translate it too when i have some time to spare. 
:)

**Comment by LonwoLonwo** [comment]
Closed as a duplicate of https://github.com/dbeaver/dbeaver/pull/21498

**Pull request closed without merging**